src/style.css

```diff
@@ -1 +1,47 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src:
+    url(fonts/Avenir-Book.ttf) format(ttf),
+    url(fonts/Avenir-Heavy.ttf) format(ttf),
+    url(fonts/Avenir.ttc) format(ttc);
```

Comment:
The format specified for the font files should be written in quotes and should be a valid format. The correct formats are 'truetype' for ttf files and 'opentype' for ttc files. Update the `format()` function accordingly.